pkg/systemd/services/service-details.jsx

```diff
@@ -403,7 +404,7 @@ export class ServiceDetails extends React.Component {
 
             // Reload after unit/timer removal
             return Promise.all(deletions).then(() =>
-                systemd_client[this.props.owner].call(SD_OBJ, SD_MANAGER, "Reload", null)
+                systemd_client[this.props.owner].call(s_bus.O_MANAGER, s_bus.I_MANAGER, "Reload", null)
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18445-20230303-111339-53caf4c9-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)